tsconfig.json

```diff
@@ -1,12 +1,17 @@
 {
     "compilerOptions": {
-        "module": "amd",
+        "module": "umd",
         "sourceMap": false,
         "rootDir": "./scripts"
     },
     "filesGlob": [
         "./scripts/**/*.ts",
         "!**/*.tests.ts",
         "./typings/index.d.ts"
+    ],
+    "files": [
```

Comment:
I think these should not be here, any tool that supports `filesGlob` should be able to expand them